packages/deltaker-flate-common/components/forslag/AvsluttDeltakelseForslagDetaljer.tsx

```diff
@@ -0,0 +1,44 @@
+import { BodyLong, Detail, Heading, HStack, Tag } from '@navikt/ds-react'
+import { EndreDeltakelseType } from '../../../../apps/nav-veileders-flate/src/api/data/endre-deltakelse-request.ts'
+import { EndringTypeIkon } from '../../../../apps/nav-veileders-flate/src/components/tiltak/EndringTypeIkon.tsx'
+import { AktivtForslag, AvsluttDeltakelseForslag } from '../../model/forslag.ts'
+import {
+  getForslagEndringAarsakText,
+  getForslagStatusTypeText
+} from '../../utils/displayText.ts'
+import { formatDateFromString } from '../../utils/utils.ts'
+
+interface Props {
+  forslag: AktivtForslag
+  avsluttDeltakelseForslag: AvsluttDeltakelseForslag
+}
+
+export const AvsluttDeltakelseForslagDetaljer = ({
```

Comment:
Alt her unntat det som står som `BodyLong` er vel helt likt `ForlengDeltakelseForslagDetaljer` og det kommer nok til å fortsette slik for de andre også? Om man har en komponenet som tar imot en `tittel` og en `EndreDeltakelseType` og en `children` så kan vel dette gjenbrukes.

Reply:
Det var lurt. Det ble mye ryddigere også! Trengte egentlig bare å sende inn selve endringen, resten kunne jeg finne enkelt på andre måter.